mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentDAOV2.java

```diff
@@ -47,15 +48,85 @@
 import com.google.common.base.Preconditions;
 
 public class CassandraAttachmentDAOV2 {
+    public static class DAOAttachmentModel {
+        private final AttachmentId attachmentId;
+        private final BlobId blobId;
+        private final String type;
+        private final long size;
+
+        private DAOAttachmentModel(AttachmentId attachmentId, BlobId blobId, String type, long size) {
```

Comment:
Why so much context, we already are in the scope of CassandraAttachmentDao.
I would rather call it Attachment.

Reply:
**Attachment** will lead to name clashes hence to FQDN. What a pitty! Moreover, it is not an attachment, I would find such a name crazily confusing... (Why? WHat? Transform an attachment to an attachment???) **DAOAttachment** seems better IMO.

Reply:
I think he means you can name it Attachment, and reference it simply by CassandraAttachmentDao.Attachment. But DAOAttachment is also fine for me.